File: Primenum.rb
print "n= "
n = gets.chomp.to_i

for a in 2..(n - 1)
  k = 0
  for i in 2..(a / 2)
    if (a % i == 0)
      k = k + 1
    end
  end
  if k <= 0
    puts a
  end
end

File: task-03/Primenum.ex
IO.puts("n = ")
n = IO.gets("")

for a <- 2..(n - 1) do
	k = 0
	for i{}< - 2..(div(a, 2) + 1) do
		if rem(a, i) == 0 do
			k= k + 1
		end
	end
	if k <= 0 do
		IO.puts(a)
	end
end
